docs/resources/ims_ecs_system_image.md

```diff
@@ -0,0 +1,88 @@
+---
+subcategory: "Image Management Service (IMS)"
+layout: "huaweicloud"
+page_title: "HuaweiCloud: huaweicloud_ims_ecs_system_image"
+description: |-
+  Manages an IMS system image resource created from ECS instance within HuaweiCloud.
+---
+
+# huaweicloud_ims_ecs_system_image
+
+Manages an IMS system image resource created from ECS instance within HuaweiCloud.
+
+## Example Usage
+
+### Creating a system image from an existing ECS instance
+
+```hcl
+variable "name" {}
+variable "instance_id" {}
+
+resource "huaweicloud_ims_ecs_system_image" "test" {
+  name        = var.name
+  instance_id = var.instance_id
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `region` - (Optional, String, ForceNew) Specifies the region in which to create the resource.
+  If omitted, the provider-level region will be used. Changing this parameter will create a new resource.
+
+* `name` - (Required, String) Specifies the name of the image.
```

Comment:
Please add detailed descriptions of fields according to the document.
https://support.huaweicloud.com/intl/en-us/api-ims/ims_03_0901.html